src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -220,9 +230,10 @@ void damped_harmonic_impl(
     }
   }
 
-  const double d0_roll_on =
-      DampedHarmonicGauge_detail::time_deriv_of_roll_on_function(
-          time, rollon_start_time, rollon_width);
+  [[maybe_unused]] const double d0_roll_on =
```

Comment:
can you put this inside the `if constexpr` below and avoid the ternary and the `[[maybe_unused]]`?

Reply:
The variable then cannot be `const` because if it is defined inside the `if constexpr` its scope is limited. I just checked that explicitly because I wasn't sure what the scoping rules were for `if constexpr`. I guess I could make the variable non-`const`, but I find that makes code more difficult to read because then I go looking for where the variable is changed. Thoughts?

Reply:
Ah, interesting -- I didn't realize the scoping restrictions of the `if constexpr`. I suppose I have a weak preference for ensuring the unnecessary variable doesn't get compiled in for the false branch of the `if constexpr` over precise const-correctness, but because this is a 'lesser of two evils' case, I'm happy to leave it up to you whether you want to take that suggestion; I'll approve either way.

Reply:
Since it's unused, I'm hoping the compiler removes it during optimizations... But yea, it's a bit of an inconvenient situation. I personally prefer `const` variables :shrug: